src/components/Entry/Entry.js

```diff
@@ -30,7 +30,7 @@ function Entry ({id, title, artists, date, notes, image, rating, masterId}) {
         <div className='entry-info'>
           <p className='entry-title'>{title}</p>
           <p className='entry-artist'>{artists}</p>
-          {rating>0 && <Rating initialValue={rating} readonly={true} size='12'/>}
+          {rating>0 && <Rating className='rating' initialValue={rating} readonly={true} allowFraction={true} size='12'/>}
```

Comment:
whats this line of code doing that you input, for clarity?

Reply:
it's the allowFraction={true}

I forgot to add it the first go around so it was only ever showing whole stars even when you submitted halves, this fixes  that